satellite/satellitedb/dbx/satellitedb.dbx

```diff
@@ -128,6 +128,13 @@ model node (
 	field free_bandwidth int64 ( updatable )
 	field free_disk      int64 ( updatable )
 
+ 	field major             int64 ( updatable )
+	field minor             int64 ( updatable )
+	field patch             int64 ( updatable )
```

Comment:
i think  major, minor, patch should just be combined into a version string

Reply:
hmm, i guess that makes it hard to sort in the table.  nvm

Reply:
We figured it out and it works great now!

Reply:
yep, @brimstone's conversation convinced me this should be a string also

Reply:
oh wait the query dang it. okay, leave this as separate fields but i still think the protobuf should be a string